packetinjector/forge.go

```diff
@@ -43,7 +43,7 @@ type ForgedPacketGenerator struct {
 	srcIP, dstIP   net.IP
 }
 
-func forgePacket(packetType string, layerType gopacket.LayerType, srcMAC, dstMAC net.HardwareAddr, srcIP, dstIP net.IP, srcPort, dstPort int64, ID int64, data string) ([]byte, gopacket.Packet, error) {
+func forgePacket(packetType string, layerType gopacket.LayerType, srcMAC, dstMAC net.HardwareAddr, srcIP, dstIP net.IP, srcPort, dstPort, TTL int64, ID int64, data string) ([]byte, gopacket.Packet, error) {
```

Comment:
nit: I would rather add `TTL` before the `srcIP, dstIP ..` in the relative position it has in the IPv4 header:

https://en.wikipedia.org/wiki/IPv4